@kiva/kv-shop/src/basketItems.ts

```diff
@@ -28,12 +30,29 @@ export async function setTipDonation({ amount, apollo }: SetTipDonationOptions)
 				}
 			}`,
 			variables: {
-				price: numeral(amount).format('0.00'),
+				price: donationAmount,
 				basketId: getBasketID(),
 			},
 		});
 		if (result?.error || result?.errors?.length) {
 			error = result?.error ?? result?.errors?.[0];
+			result?.errors?.forEach((err: any) => {
```

Comment:
`any` should be something else here. If there isn't a built in type, you can create one custom in this file, based on the props we expect.

Reply:
That could work. You could also inline it with `(result?.errors as GraphQLErrors ?? []).forEach`

Reply:
I was thinking the same, but I wasn't sure it was possible, thanks!